components/802.15.4_RF/mcr20a-rf-driver/.gitignore

```diff
@@ -0,0 +1,6 @@
+*~
```

Comment:
Same here

Reply:
These are verbatim copy of the existing repositories, but maybe you are right. We should clean up the stuff that we don't need.